src/services/cssCompletion.ts

```diff
@@ -918,15 +918,15 @@ export class CSSCompletion {
 		return result;
 	}
 
-	private getEntryDescription(entry: languageFacts.IEntry2): string | MarkupContent {
-		const rawDescription = languageFacts.getEntryDescription(entry);
-		if (!this.doesSupportMarkdown() && typeof rawDescription !== 'string' && rawDescription && rawDescription.kind === 'markdown') {
+	private getEntryDescription(entry: languageFacts.IEntry2): string | MarkupContent | undefined {
+		if (this.doesSupportMarkdown()) {
 			return {
-				kind: 'plaintext',
-				value: rawDescription.value
+				kind: 'markdown',
```

Comment:
Let's do it like in the html-ls: 'generateMarkupcontent'. It can generate the MarkupContent or string (based on capabilities)